.gitignore

```diff
@@ -9,7 +9,7 @@ jshint.log
 docs/app/assets/licenses.txt
 src/fonts/*-icons.woff
 src/fonts/*-icons.ttf
-src/styles/hpe-icons.less
+src/styles/hpe-icons.css
```

Comment:
@donna-marijne Do you think this file be included in our repo? The ux-icon.less file is (https://github.com/UXAspects/UXAspects/blob/develop-2.0.4/src/styles/ux-icons.less) and I don't know how many times I have been asked when someone first clones the repo and tries to run the app why `hpe-icons.less` is missing (because they haven't run grunt iconset).

Thoughts?

Reply:
Yeah I think it's not needed, I have removed it in the selenium branch already because my environment cannot build the iconset correctly and it breaks the e2e tests.

Reply:
Yes I agree, I've removed it.